yogstation/code/modules/events/immovable_duck.dm

```diff
@@ -32,6 +32,7 @@
 /obj/effect/immovablerod/duck/Initialize()
 	. = ..()
 	SpinAnimation(24,-1)
+	SSaugury.unregister_doom(src)
```

Comment:
Instead, make a variable that can be set by mappers on immovable rods to make it not auto follow

Reply:
Sure, if i knew how.

Reply:
just add var to immovable rod called something like notdebris and then on immovable rod initialise if(notdebris == true ){SSaugury.unregister_doom(src)}